Bio/PDB/Polypeptide.py

```diff
@@ -197,8 +197,11 @@ def get_ca_list(self):
         """
         ca_list = []
         for res in self:
-            ca = res["CA"]
-            ca_list.append(ca)
+            try:
```

Comment:
I'm not sure what you mean.
I try to do "ca = res.get("CA")" instead and I got

    ca = res.get("CA")
AttributeError: 'Residue' object has no attribute 'get'

So what do you mean by "dict.get syntax" ?